exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/cloud_store_config.ipynb

```diff
@@ -11,27 +11,13 @@
     "On the high level it does the following:\n",
```

Comment:
<ul><li>later, I think this whole block should go into the notebook connector</li><li>and I think, we need to introduce a namespace for the notebook connector, because we are cluttering the exasol namespace </li></ul><p> </p><p>Please create tickets for both.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
This is not just for cloud-store, I assume, but also for `create_schema` functions, for example. Right?

Reply:
https://github.com/exasol/notebook-connector/issues/77

Reply:
https://github.com/exasol/data-science-sandbox/issues/135

Reply:
> This is not just for cloud-store, I assume, but also for `create_schema`-like functions, for example. Right?

likely